perfkitbenchmarker/linux_benchmarks/iperf_benchmark.py

```diff
@@ -73,23 +93,31 @@ def Prepare(benchmark_spec):
   """
   vms = benchmark_spec.vms
   if len(vms) != 2:
-    raise ValueError(
-        'iperf benchmark requires exactly two machines, found {0}'.format(len(
-            vms)))
+    raise ValueError(f'iperf benchmark requires exactly two machines, found {len(vms)}')
 
   for vm in vms:
     vm.Install('iperf')
     if vm_util.ShouldRunOnExternalIpAddress():
-      vm.AllowPort(IPERF_PORT)
-    stdout, _ = vm.RemoteCommand(('nohup iperf --server --port %s &> /dev/null'
-                                  '& echo $!') % IPERF_PORT)
-    # TODO store this in a better place once we have a better place
-    vm.iperf_server_pid = stdout.strip()
+      if TCP in FLAGS.iperf_benchmarks:
+        vm.AllowPort(IPERF_PORT)
+      if UDP in FLAGS.iperf_benchmarks:
+        vm.AllowPort(IPERF_UDP_PORT)
+    if TCP in FLAGS.iperf_benchmarks:
+      stdout, _ = vm.RemoteCommand(f'nohup iperf --server --port {IPERF_PORT}'
+                                   ' &> /dev/null & echo $!')
+
+      # TODO store this in a better place once we have a better place
+      vm.iperf_tcp_server_pid = stdout.strip()
+    if UDP in FLAGS.iperf_benchmarks:
+      stdout, _ = vm.RemoteCommand(f'nohup iperf --server -u --port {IPERF_UDP_PORT}'
```

Comment:
`--udp` for better readability?

Reply:
general comment for all flags `-e`, `-b`, etc.

Reply:
switched to full length flags for readability